packages/realm/scripts/submit-analytics.mjs

```diff
@@ -105,7 +104,9 @@ function getProjectRoot() {
  */
 function getPackageJson(packagePath) {
   const packageJson = path.resolve(packagePath, "package.json");
-  return fse.readJsonSync(packageJson);
+  const buffer = fs.readFileSync(packageJson);
+  const asJson = JSON.parse(buffer);
```

Comment:
1. Won't this throw if you try to parse a Buffer rather than a string?
   * I believe you could provide an `{ encoding: ".." }` 2nd arg to `readFileSync` in order to get a string back (see [various encodings](https://github.com/nodejs/node/blob/41f70568a46efceb59bdb00a727b4fe8d858cb2b/lib/buffer.js#L719)).
3. Maybe `asObject` is a more suitable name? Or perhaps return the result directly, your call 🙂